From: Joel DeJesus <dejesus.joel@e-flamingo.jp>
Date: Sun, 30 Apr 2017 19:24:57 +0900
Subject: setting password to test tor control code

---
 t/01-network.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/01-network.t b/t/01-network.t
index dd09557..f46b6d4 100644
--- a/t/01-network.t
+++ b/t/01-network.t
@@ -13,7 +13,7 @@ BEGIN { use_ok('Network::Tor') };
 ######################### set up event loop #######################
 
 # create tor control object
-my $torcontrol = Network::Tor->new($ENV{'TorControlPassword'},$ENV{'TorControlAddress'});
+my $torcontrol = Network::Tor->new('ghost1',"127.0.0.1:9051");
 $torcontrol->connect();
 
 # create event loop
